tests/lib/rules/prop-types.js

```diff
@@ -2335,6 +2352,30 @@ ruleTester.run('prop-types', rule, {
       `,
       parser: parsers.BABEL_ESLINT
     },
+    {
+      // issue #2138
+      code: `
+        type UsedProps = {|
+          usedProp: number,
+        |};
+
+        type UnusedProps = {|
+          unusedProp: number,
+        |};
+
+        type Props = {| ...UsedProps, ...UnusedProps |};
+
+        function MyComponent({ usedProp }: Props) {
+          return <div>{usedProp}</div>;
+        }
+      `,
+      parser: parsers.BABEL_ESLINT,
+      errors: [{
+        message: "'notOne' is missing in props validation",
```

Comment:
possibly a dumb question, but why is this erroring? `notOne` doesn't appear anywhere else in this test?

Reply:
@ghiculescu it is at line 4791

Reply:
@moroine I'm not sure I understand. Why is a separate test causing this test to fail? Aren't they each independent tests?